+++ devour.c	2020-08-23 20:40:49.965458610 +0600
@@ -11,6 +11,7 @@
    int is_unsafe = 0;
    char cmd[512] = {0};
 
+   strcat(cmd, "$SHELL -i -c '");
    while (*++argv) {
       if (!strcmp(*argv, "--")) {
          is_unsafe = 1;
@@ -19,6 +20,7 @@
       strcat(cmd, *argv);
       strcat(cmd, is_unsafe && *(argv + 1) ? "\\ " : " ");
    }
+   strcat(cmd, "> /dev/null 2>&1; exit'");
    system(cmd);
 }
 
